### ISSUE TITLE
_SAPS4_Opt_06 Read Trace not 100% accurate

### ISSUE THREAD

**Issue by joegoerlich** [SAP-samples/s4hana-hana-audit-policies#13]
When creating the AUDIT POLICY "_SAPS4_Opt_06 Read Trace" in a tenant DB using the given statement, this results in a partially invalid audit policy as the schema SYS_DATABASES does only exist in the system DB.

**Comment by saint-j** [comment]
thanks for the suggestion. Give me a bit time to incorporate your ideas.

**Comment by saint-j** [comment]
you are right - the part of the policy regarding the _SYS_DATABASE object is inactive.
I'm reluctant to create specific policies for SYSTEM DB and Tenants. I added a comment in pull request #17
